idaes/ui/fsvis/static/js/paper.js

```diff
@@ -63,6 +63,28 @@ export class Paper {
         // context menu
         self._paper.on("element:contextmenu", function(cellView, evt) {
             cellView.model.rotate(90)
+            // This is needed to keep the text labels for the unit models in the correct orientation
+            // x and y were specifically picked to keep the label in the same place 
+            // in relation to the unit model (bottom middle)
+            let x = 0
+            let y = 0
+            if (cellView.model.angle() == 0) {
```

Comment:
Since it is used a few times, would it make sense to call `cellView.model.angle()` once and save the result to a `const`?

Reply:
Good point